Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:latest
+FROM ubuntu:latest
```

Comment:
This was causing file permission issues when you try to run the binary. Switching to ubuntu fixed it.